string baseLat
string baseLon
string roverLat
string roverLon
string tennisLat
string tennisLong
